### ISSUE TITLE
Filtering 

### ISSUE THREAD

**Issue by mg262** [osm2vectortiles/osm2vectortiles#469]
The more I look at OSM2VectorTiles the neater it looks.  Unfortunately, the standard vector tiles are too detailed (and so too large) for my use case -- in particular, I don't need the buildings.  What's the easiest way of filtering out the data I don't need when creating custom tiles?  Should I filter the OSM extract, or can I filter during the import process?

**Comment by lukasmartinelli** [comment]
@klokan has some ideas to remove PBF layers from MBTiles.
But I agree that the tiles are too fat right now!

**Comment by mg262** [comment]
It wasn't a criticism -- I need particularly compact tiles as they will be used on tablets.  Am I right in thinking that I can drop the filesize a lot by tinkering with https://github.com/osm2vectortiles/osm2vectortiles/tree/master/osm2vectortiles.tm2source/data.yml to remove the things I don't need?  Is that the easiest approach?

**Comment by lukasmartinelli** [comment]
> It wasn't a criticism -- I need particularly compact tiles as they will be used on tablets. Am I right in thinking that I can drop the filesize a lot by tinkering with https://github.com/osm2vectortiles/osm2vectortiles/tree/master/osm2vectortiles.tm2source/data.yml to remove the things I don't need? Is that the easiest approach?

Yes that will be easier in v3 where you can pick which layers you want. Sorry I always have to point to the future like a politician I hope we can soon release this finally public.

**Comment by sfkeller** [comment]
> Sorry I always have to point to the future like a politician. 
> I hope we can soon release this finally public.

LOL! This is not only a politicians attitude - but also part of "definition of done" - and a good practice for community moderation :-)
